crates/holochain/src/test_utils/network_simulation.rs

```diff
@@ -315,7 +316,7 @@ fn get_cached() -> Option<GeneratedData> {
             .ok()
             .flatten()?;
         let ops = get_ops(&mut txn);
-        let peer_data = txn.p2p_list_agents().unwrap();
+        let peer_data = txn.p2p_list_agents(Arc::new(KitsuneSpace::new(vec![0; 36]))).unwrap();
```

Comment:
This should be fine I think, it's just a cache key which was previously the hard-coded string to the test database